src/manual.md

```diff
@@ -0,0 +1,2706 @@
+Design](/manual/system-design), provides the details you will need to interact
+with Taskcluster. Finally, [Using Taskcluster](/manual/using/) presents a
+collection of common use cases and approaches to satisfy them.
+
+---
+filename: tasks/index.md
+title: Tasks
+order: 10
+---
+
+This chapter focuses on Taskcluster's core concept: tasks.
+
+At the highest level, Taskcluster executes tasks. New tasks are added to one of
+several queues, and workers consume tasks from those queues and execute them.
+In the process, these services send [Pulse](/manual/design/apis/pulse) messages
+about task status, and attach the results of the task -- including logs and
+output artifacts -- to the task itself.
+
+A task is described by a task definition, which is a simple data structure
+usually encoded in JSON or YAML.  Each task has a unique `taskId`, assigned
+randomly by the caller when the task is created.  The full schema for the task
+definition is given in [the reference
+documentation](/reference/platform/taskcluster-queue/docs/task-schema), but a
+simple task might look like this:
+
+```yaml
+provisionerId:      aws-provisioner-v1
+workerType:         tutorial
+created:            2020-01-04T04:18.084Z
+deadline:           2020-01-05T04:18.084Z
+metadata:
+  name:             Example Task
+  description:      Eample from TC Manual
+  owner:            nobody@taskcluster.net
+  source:           https://github.com/taskcluster/Taskcluster-docs
+payload:
+  image:            ubuntu:16.04
+  command:          ['echo', 'hello world']
+```
+
+You can use the [task creator](https://tools.taskcluster.net/task-creator) to
+experiment with creating tasks.
+
+The next few sections describe the many powerful features of tasks, while the
+[next chapter](/manual/task-execution) covers their execution by workers.
+
+---
+filename: tasks/times.md
+title: Times
+order: 10
+---
+
+Each task has three associated times:
+* `created` is the time the task was created;
+* `deadline` is the time by which the task must be completed; and
+* `expires` is the time after which all record of the task and its runs will be deleted.
+
+The deadline exists to ensure that tasks do not remain pending forever. It is
+usually set to one day after `created`, on the assumption that running the task
+more than a day later is not useful.
+
+Task expiration ensures that tasks are not stored indefinitely, thereby
+controlling storage costs. Expiration is often set to one year after creation,
+but for common tasks, tasks which produce large artifacts, or tasks which are
+unlikely to be useful later, a much shorter expiration is appropriate.
+
+---
+filename: tasks/workertypes.md
+title: Worker Types
+order: 20
+---
+
+Workers execute tasks, but there are many types of workers. A task specifies a
+single "worker type" by which it should be executed, using the two-part
+namespace `<provisionerId>/<workerType>`.  Workers with the same worker type
```

Comment:
I think I was careful to use "worker type" to refer to the whole thing, and `workerType` to refer to the task property, if that helps (probably not..)